### ISSUE TITLE
A11y issue: Kraken theme error message text color is not accessible

### ISSUE THREAD

**Issue by pasanchMSFT** [microsoft/iot-cardboard-js#795]
**Describe the bug**
Kraken theme has `#9C2334` set as the error message text color. This color does not contrast well with the background which is usually `#18203D`. This is not accessible.

**To Reproduce**
1. Go to 3D scenes and select Kraken as your theme
2. Trigger an error message

**Expected behavior**
Error message text color contrast with the background of the app should be accessible.

**Screenshots**
![image](https://user-images.githubusercontent.com/55852250/206052345-58151ec2-8a47-4c6e-8d8f-1bbb0cb16348.png)

**Additional context**
To check if color contrast is accessible check in this site: https://www.accessibilitychecker.org/color-contrast-checker/
